controller/network/options.go

```diff
@@ -31,6 +31,7 @@ const (
 	DefaultNetworkOptionsPendingLinkTimeout      = 10 * time.Second
 	DefaultNetworkOptionsSmartRerouteFraction    = 0.02
 	DefaultNetworkOptionsSmartRerouteCap         = 4
```

Comment:
We already have `DefaultNetworkOptionsRouteTimeout` above, I think we should probably use that

Reply:
Sure! Wan't sure if we wanted more granularity here